### ISSUE TITLE
xdg-open: flags '-h' and '--help' in test but 'help' missing in 'getopt --long'

### ISSUE THREAD

**Issue by like2omg** [termux/termux-packages#801]
I'm sorry ahead of time if this belongs in termux-api but I searched the root "owner" but could not find the file's location (only references to launching it). I also have a habit to check for package updates so I did before knowing the app updated (I apologize if I broke anythin with that, or with `apt-get clean`).

The file on my device is located at `/data/data/com.termux/files/usr/bin/xdg-open`. The file refuses to attach (someone on Twitter mentioned "AWS" is having issues, and also talked about Github as well), so I cannot provide the full file, so I'll provide context.

The argument parser `case "$1" in` section has the following:
```sh
-h | --help) show_usage;;
```

Validation of that is only one of those two (`-o` is fine, `--long` is the issue):
```sh
TEMP=`busybox getopt \
     -n $SCRIPTNAME \
     -o h \
     --long send,view,chooser,content-type:,\
     -- "$@"`
eval set -- "$TEMP"
```

Note that I can only understand `getopt` a litte bit, yet I know what change is needed. (Is there supposed to be a trailing `,`? I understand if that is how `--` works, yet I have not looked at any dos / manpages for `getopt` in a while.)
```sh
     --long send,view,chooser,content-type:,help,\
```

Replace trailing `,` before trailing `\` with space if necessary.

**Comment by fornwall** [comment]
Thanks a lot! This has been fixed in version 0.39 of the termux-tools package, update to it with `packages upgrade`.

**Comment by like2omg** [comment]
You're welcome @fornwall . Sorry I didn't respond sooner.
